### PR TITLE
Fix reference in FAQ

### PR DESCRIPTION
**Pull request by atdt**

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -146,9 +146,9 @@ really desperate (e.g. if you have no completed backup of that file and you'ld
 rather get a partial file extracted than nothing). You do **not** want to give
 that option under any normal circumstances.
 
-Note that checkpoints inside files are created only since version 1.1, 
+Note that checkpoints inside files are created only since version 1.1
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: did you intentionally remove the comma? why?

File: docs/faq.rst [hunk 2]
```diff
@@ -146,9 +146,9 @@ really desperate (e.g. if you have no completed backup of that file and you'ld
 rather get a partial file extracted than nothing). You do **not** want to give
 that option under any normal circumstances.
 
-Note that checkpoints inside files are created only since version 1.1, 
+Note that checkpoints inside files are created only since version 1.1 
 make sure you have an up-to-date version of borgbackup if you want to continue instead of retransferring a huge file.
-In some cases, there is only an outdated version shipped with your distribution (e.g. Debian). See :ref:`_installation`
+In some cases, there is only an outdated version shipped with your distribution (e.g. Debian). See :ref:`installation`.
```
Comment (ThomasWaldmann) [inline docs/faq.rst]: good catch, yeah that link was broken and also the full stop was missing.

File: docs/faq.rst (hunk 1 above)
Comment (atdt) [reply docs/faq.rst]: By mistake; sorry. Not used to editing files using the GitHub UI.

**Comment by ThomasWaldmann** [comment]
thanks! this is now in master branch - would you like to backport this to 1.1-maint branch also, so it gets into borg 1.1.10 docs?

**Pull request merged**